gap/SemilinearMatrixGroups.gi

```diff
@@ -194,3 +194,85 @@ function(d, q, s)
     # conjugate the result so that it preserves the standard unitary form 
     return ConjugateToStandardForm(MatrixGroupWithSize(F, generators, size), "U");
 end);
+
+# Construction as in Proposition 6.4 of [HR05]
+BindGlobal("SymplecticSemilinearSp",
+function(d, q, s)
+    local F, gammaL1, As, Bs, m, omega, AandB, C, i, range, generators, size;
+    if d mod s <> 0 or not IsPrime(s) then
+        ErrorNoReturn("<s> must be prime and a divisor of <d> but <s> = ", s,
+                      " and <d> = ", d);
```

Comment:
```suggestion
        ErrorNoReturn("<s> must be prime and a divisor of <d>");
```
I don't think printing `s` and `d` is necessary here, as the user can just query them via the break loop once they hit that error, no?

Reply:
Well, so far, we always printed the relevant arguments in error messages - so you're suggesting to alter this paradigm?

Reply:
Yes, I think I also suggested this in other PRs before. i can also turn this into an issue if you prefer. The way I suggest is the usual way to do this in GAP